antarest/utils.py

```diff
@@ -9,265 +9,18 @@
-) -> MatrixGarbageCollector:
-    if study_service and matrix_service:
-        return MatrixGarbageCollector(
-            config=config,
-            study_service=study_service,
-            matrix_service=matrix_service,
-        )
-    else:
-        _, _, _, _, _, matrix_service, study_service = create_core_services(app_ctxt, config)
-        return MatrixGarbageCollector(
-            config=config,
-            study_service=study_service,
-            matrix_service=matrix_service,
-        )
-
-
-def create_archive_worker(
-    config: Config,
-    workspace: str,
-    local_root: Path = Path("/"),
-    event_bus: Optional[IEventBus] = None,
-) -> AbstractWorker:
-    if not event_bus:
-        event_bus, _ = create_event_bus(None, config)
-    return ArchiveWorker(event_bus, workspace, local_root, config)
-
-
-def create_simulator_worker(
-    config: Config,
-    matrix_service: MatrixService,
-    event_bus: Optional[IEventBus] = None,
-) -> AbstractWorker:
-    if not event_bus:
-        event_bus, _ = create_event_bus(None, config)
-    return SimulatorWorker(event_bus, matrix_service, config)
-
-
-def create_services(config: Config, app_ctxt: Optional[AppBuildContext], create_all: bool = False) -> Dict[str, Any]:
-    services: Dict[str, Any] = {}
-
-    (
-        cache,
-        event_bus,
-        task_service,
-        file_transfer_manager,
-        user_service,
-        matrix_service,
-        study_service,
-    ) = create_core_services(app_ctxt, config)
-
-    maintenance_service = build_maintenance_manager(app_ctxt, config=config, cache=cache, event_bus=event_bus)
-
-    launcher = build_launcher(
-        app_ctxt,
-        config,
-        study_service=study_service,
-        event_bus=event_bus,
-        task_service=task_service,
-        file_transfer_manager=file_transfer_manager,
-        cache=cache,
-    )
-
-    watcher = create_watcher(config=config, app_ctxt=app_ctxt, study_service=study_service)
-    services["watcher"] = watcher
-
-    if config.server.services and Module.MATRIX_GC.value in config.server.services or create_all:
-        matrix_garbage_collector = create_matrix_gc(
-            config=config,
-            app_ctxt=app_ctxt,
-            study_service=study_service,
-            matrix_service=matrix_service,
-        )
-        services["matrix_gc"] = matrix_garbage_collector
-
-    if config.server.services and Module.AUTO_ARCHIVER.value in config.server.services or create_all:
-        auto_archiver = AutoArchiveService(study_service, config)
-        services["auto_archiver"] = auto_archiver
-
-    services["event_bus"] = event_bus
-    services["study"] = study_service
-    services["launcher"] = launcher
-    services["matrix"] = matrix_service
-    services["user"] = user_service
-    services["cache"] = cache
-    services["maintenance"] = maintenance_service
-    return services
+def to_json(data: t.Any, indent: t.Optional[int] = None) -> bytes:
```

Comment:
What would you think about moving this to a `core.serialization` module ? Something more targetted than the plain "utils".

Also we should add a docstring to mention that we do this to benefit from the performances of pydantic serialization